Samples and instructions for monitoring in OpenShift

matrober-uk opened the pull request:
Please ensure all items are complete before opening.

- [x] Tick to sign-off your agreement to the [IBM Contributor License Agreement](https://github.com/ibm-messaging/mq-metricsamples/CLA.md)
- [ ] N/A You have added tests for any code changes
- [x] You have updated the [CHANGELOG.md](https://github.com/ibm-messaging/mq-metric-samples/CHANGELOG.md)
- [x] You have completed the PR template below:

## What
Added sample config files and instructions to demonstrate deploying to OpenShift

## How
Added new directory with sample OpenShift deployer, pod and service definitions for easy reuse

## Testing
Validated by deployment to OpenShift cluster in IBM Cloud as part of creation of detailed blog post

## Issues
N/A

ibmmqmet commented:
Just as in the word doc, the README is not making it clear that this is designed as an addition to the inbuilt collector. It's therefore not "for openshift" (which I'd expect from the directory name) but for a CP4I system. In this repo, I'd expect the samples to be for a standalone solution - the settings of usePubs, useObjectStatus etc.

matrober-uk commented:
Hi @ibmmqmet - I've added an additional comment to this PR to address your comments;

- Renamed the directory from "openshift" to "cp4i"
- Enhanced the description to describe that queue monitoring is not currently supported by the MQ Certified Container, so the second-monitor pod approach is a practical workaround until the Certified Container is enhanced to support that use case (which is not imminent)

Ready for your re-review please.

matrober-uk commented:
thanks Mark for merging  👍
